### PR TITLE
Draft: chore: Add playground

### PR DESCRIPTION
**Pull request by sandrina-p**
wip

---

[Linear issue (internal)](https://linear.app/remote/issue/RMT-494/jsf-create-a-playground-in-the-repo)

### CODE REVIEW

File: playground/.eslintrc.cjs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+module.exports = {
```
Comment (sandrina-p) [inline playground/.eslintrc.cjs:1]: @gabrielse, I've cherry-picked your commit for #18. Let's talk about the Playground here. I know that @brennj also had some ideas in mind for it, so please coordinate among yourselves, okay? :)

**Comment by gabrielseco** [comment]
Hey @brennj,

I was thinking on adding a playground for json-schemas.

Do you have anything in mind, I started the other day by adding a vite react starter with the content of this [codesandbox](https://codesandbox.io/s/json-schema-form-demo-react-igh3sk?from-embed).

I was thinking about it and I think it could be more useful to have the playground that we have in [storybook](https://json-schema-form.vercel.app/?path=/docs/playground--playground)

Let me know what ideas do you have for this, in case they're different.

**Comment by brennj** [comment]
> Hey @brennj,
> 
> I was thinking on adding a playground for json-schemas.
> 
> Do you have anything in mind, I started the other day by adding a vite react starter with the content of this [codesandbox](https://codesandbox.io/s/json-schema-form-demo-react-igh3sk?from-embed).
> 
> I was thinking about it and I think it could be more useful to have the playground that we have in [storybook](https://json-schema-form.vercel.app/?path=/docs/playground--playground)
> 
> Let me know what ideas do you have for this, in case they're different.

Hey @gabrielseco I'm really sorry for the delay in answering this! I'm really open to whatever you think is a good judgement call here. I haven't had a chance to fully dedicate looking into this properly since i'm heads deep in JSON logic. Your approach sounds good so far though!

**Comment by sandrina-p** [comment]
I'm closing this PR until someone pick it up again.

**Pull request closed without merging**